src/MainWindow.cpp

```diff
@@ -610,7 +610,7 @@ void MainWindow::MessageReceived(BMessage *msg)
 			}
 			
 			BRect r(Frame());
-			ReconcileWindow *recwin = new ReconcileWindow(BRect(100,100,600,425), acc);
+			ReconcileWindow *recwin = new ReconcileWindow(BRect(100,100,650,425), acc);
```

Comment:
Let's leave it alone. (See below.)

Reply:
So should I change it back to 600 or should I leave it at 650?

Reply:
Change it back.